docs/testing/troubleshoot-manifest.md

```diff
@@ -1,13 +1,16 @@
 ---
-title: Validate and troubleshoot issues with your manifest
-description: Use these methods to validate the Office Add-ins manifest.
-ms.date: 11/26/2019
+title: Validate an Office Add-in's manifest
+description: Learn how to validate an Office Add-in's manifest.
```

Comment:
Consider adding some methods to this description to boost SEO. Like, "Learn how to validate an Office Add-in's manifest using the XML schema and other tools."